### PR TITLE
Use `sprint`

### PR DESCRIPTION
**Pull request by fonsp**

### CODE REVIEW

File: test/prints.jl [hunk 1]
```diff
@@ -20,15 +18,13 @@
...
-        io = IOBuffer()
         app = App()
         add_post!(app, "test") do req
             body = req.body
             return body
         end
 
-        show(io, app)
-        iostr = String(take!(io))
+        iostr = sprint(show, app)
```
Comment (fonsp) [inline test/prints.jl:27]: Or even shorter:
```julia
iostr = repr(app)
```

Comment (eliascarv) [reply test/prints.jl:27]: I think `str = repr(app)` is even better!

**Comment by eliascarv** [comment]
Hi @fonsp! Thanks for helping me to develop the WebAPI.jl.
Did you read the examples in the README?
What did you think of the code design (function names, argument order, ...)?
I wanted to hear someone's opinion on this.

File: test/prints.jl (hunk 1 above)
Comment (eliascarv) [reply test/prints.jl:27]: Do you want to do the commit to make this change?
Or do you prefer that I do?

**Comment by fonsp** [comment]
Looks nice but I haven't really used it for a project yet! The best thing you can do is:
1. Start using your package in a real project! See which problems come up.
1. Look at other projects (in other programming languages), like fastapi in python, not-sure-what in NodeJS (you could look at expressJS for ideas on routing API). Which of these ideas works well in Julia?
2. Come up with some sample projects (e.g. a CRUD API for a movie watchlist, weather API, ...) and write them in your API. Try to change your API and see what your projects would look like.

Some comments from me:
- It would be nice if I could define "subroutes" in a separate function. E.g. I have one function to create routes for `/a` and `/b`, and I later "add that router" to my main app under `/hello`, hence serving `/hello/a` and `/hello/b`. I believe you can do this in express, where an *app* can hold multiple *routers*, each with a possible *preifx*, like`/hello` in my example.
- You could think about a macro that automatically turns a function like `hello(; one::Bool, two::String)` into a handler for `GET /hello?one=true&two=yes` or something similar. I think this is what fastapi does.
- Maybe you could add a function that automatically sets up [these three routes](https://github.com/eliascarv/WebAPI.jl/blob/4f923352daa7438daa1213c2992f4770c4398061/README.md?plain=1#L55-L84), with a single function call? Or maybe just two: the query GET and the json POST (the `/:a/:b/:c/` is order-dependent).

**Pull request merged**